pillarbox-demo-tv/build.gradle.kts

```diff
@@ -7,6 +7,34 @@ plugins {
     alias(libs.plugins.kotlin.serialization)
 }
 
+android {
+    val versionDimension = "version"
+    flavorDimensions += versionDimension
+    productFlavors {
+        register("prod") {
+            dimension = versionDimension
+        }
+
+        if (System.getenv("CI") == "true") {
+            register("nightly") {
+                dimension = versionDimension
+                applicationIdSuffix = ".nightly"
+                versionNameSuffix = "-nightly"
+            }
+        }
+    }
+
+    buildFeatures {
+        buildConfig = true
+    }
```

Comment:
We're not using `BuildConfig` in `pillarbox-demo-tv`, so we probably don't need that